crc32_amd64.go

```diff
@@ -1,9 +1,9 @@
-//+build !appengine,!gccgo
-
 // Copyright 2011 The Go Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
+// +build !appengine,!gccgo
```

Comment:
It's more idiomatic style and consistent with how build constraints are done.

Compare with:

- https://github.com/golang/go/blob/c1037aad4d10078b536625683364dcceed6c4e21/src/runtime/cputicks.go
- https://github.com/golang/go/blob/f78a4c84ac8ed44aaf331989aa32e40081fd8f13/src/runtime/mmap.go
- https://github.com/golang/go/blob/f78a4c84ac8ed44aaf331989aa32e40081fd8f13/src/runtime/cgo/gcc_signal_darwin_lldb.c
- https://github.com/golang/go/blob/f78a4c84ac8ed44aaf331989aa32e40081fd8f13/src/crypto/sha1/sha1block_generic.go
- https://github.com/golang/go/blob/f78a4c84ac8ed44aaf331989aa32e40081fd8f13/src/crypto/sha256/sha256block_decl.go

It's just a style change and should not have any effect. If you dislike it, I can remove it.